scripts/helpers/build.js

```diff
@@ -35,7 +35,8 @@ module.exports = (options = {}) => {
       outDir: viewOutDir,
       watch,
       // HMR seems to be broken: https://github.com/parcel-bundler/parcel/issues/2894
-      hmr: false
+      hmr: false,
```

Comment:
So we shouldn't just use an older version until it is resolved? Or maybe we can add a ticket to change it back?

Reply:
I feel like there's more value in staying on the latest version of Parcel than saving HMR. I've created a JIRA ticket for switching HMR back on once it's fixed in Parcel: https://jira.corp.adobe.com/browse/CORE-44089